src/ParkMap.js

```diff
@@ -14,7 +14,7 @@ class ParkMap extends Component {
 
   getPark = (event) => {
     this.setState({
-      selectedPark: this.props.parks.find( park => park.urlCode === event.target.options.id)
+      selectedPark: this.props.parks.find( park => park.parkName === event.target.options.id)
```

Comment:
I think we should use park.urlCode to avoid issues we might have with spaces in park.parkName. Then we can just add in some conditional logic for Sequoia on our fetch call when we pull in the data to set park.urlCode = 'seki1'.

Reply:
I'll update line 44 also so that the Marker component key is the park.urlCode also instead of the park.parkName